pkg/cmd/provisioning/constants.go

```diff
@@ -37,4 +37,8 @@ const (
         "sub"
     ]
 }`
+	// featureGateAnnotation is the annotation used to indicate that a specific manifest is hidden behind a feature gate.
+	featureGateAnnotation = "release.openshift.io/feature-gate"
+	// validTechPreviewAnnotationValue is the only valid value for the feature-gate annoation
+	validTechPreviewAnnotationValue = "TechPreviewNoUpgrade"
```

Comment:
You can use `configv1.TechPreviewNoUpgrade` instead of defining your own variable for this.

```console
cloud-credential-operator $ git --no-pager grep TechPreviewNoUpgrade
vendor/github.com/openshift/api/config/v1/types_feature.go:     // TechPreviewNoUpgrade turns on tech preview features that are not part of the normal supported platform. Turning
vendor/github.com/openshift/api/config/v1/types_feature.go:     TechPreviewNoUpgrade FeatureSet = "TechPreviewNoUpgrade"
vendor/github.com/openshift/api/config/v1/types_feature.go:     TechPreviewNoUpgrade: newDefaultFeatures().
```